tableauserverclient/server/endpoint/projects_endpoint.py

```diff
@@ -20,6 +25,20 @@ def get(self, req_options=None):
         all_project_items = ProjectItem.from_response(server_response.content)
         return all_project_items, pagination_item
 
+    def get_default(self):
+        if (Version(self.parent_srv.version) < Version("2.7")):
+            for proj in Pager(self):
+                if proj.is_default():
+                    return proj
+
+            return None
+        else:
+            ro = RequestOptions()
+            ro.filter.add(Filter("name", "eq", "Default"))
```

Comment:
Sometimes it's uppercase and sometimes it's lower case (I think pre 9.0 installs vs post) so you'll need to normalize the casing